shibboleth.php

```diff
@@ -845,7 +845,9 @@ function shibboleth_disable_login_form() {
 	if ( $disable && ! $bypass ) {
 	?>
 		<style type="text/css">
-			.login #loginform p {
+			.login #loginform p ,
+			.login #loginform .user-pass-wrap
+			{
```

Comment:
Small adjustment to align with WordPress Coding Standards for CSS
```suggestion
			.login #loginform p,
			.login #loginform .user-pass-wrap {
```